<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Celsjusz</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QDial" name="dialCelsius">
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdCelsius">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Zakres</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="minCelsius">
              <property name="minimum">
               <number>-100</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxCelsius">
              <property name="minimum">
               <number>-100</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Farenheit</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QDial" name="dialFarenheit">
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>213</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdFarenheit">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Zakres</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="minFarenheit">
              <property name="minimum">
               <number>-148</number>
              </property>
              <property name="maximum">
               <number>212</number>
              </property>
              <property name="value">
               <number>32</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxFarenheit">
              <property name="minimum">
               <number>-148</number>
              </property>
              <property name="maximum">
               <number>212</number>
              </property>
              <property name="value">
               <number>212</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="title">
         <string>ZÅ‚oty</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QDial" name="dialPLN">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdPLN">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Euro</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QDial" name="dialEuro">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdEuro">
           <property name="smallDecimalPoint">
            <bool>true</bool>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Filled</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dialEuro</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdEuro</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialPLN</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdPLN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
